Modules/Filtering/FFT/include/itkComplexToComplex1DFFTImageFilter.hxx

```diff
@@ -25,8 +25,7 @@
 namespace itk
 {
 template <typename TInputImage, typename TOutputImage>
-ComplexToComplex1DFFTImageFilter<TInputImage, TOutputImage>::ComplexToComplex1DFFTImageFilter()
-{}
+ComplexToComplex1DFFTImageFilter<TInputImage, TOutputImage>::ComplexToComplex1DFFTImageFilter() = default;
```

Comment:
This commit has an appropriate commit message. The other two are in a need of a better message. I merged Niels' PR, so rebasing on current master is in order too.